addons/patches/unscfoundries/CfgVehicles.inc

```diff
@@ -62,4 +62,78 @@ class CfgVehicles
       class Right : Right {};
     };
   };
+  class Splits_Pelican_base;
+  class Splits_UNSC_D77_TC_Pelican: Splits_Pelican_base
+  {
+    irTargetSize = 1.6; 
+    radarTargetSize = 1.0;
+    class Components: Components
+    {
+      class TransportPylonsComponent
+      {
+        class Pylons
+        {
+          class PylonRight1
+					{
+						attachment = "";
+						priority = 4;
+						hardpoints[] = {"Splits_T_Missiles"};
+						turret[] = {};
+						UIposition[] = {0.25,0.1};
+					};
+					class PylonRight2: PylonRight1
+					{
+            attachment = "";
+						priority = 3;
+						UIposition[] = {0.25,0.2};
+					};
+					class PylonRight3
+					{
+						attachment = "ace_maverick_L_PylonRack_3Rnd";
+						hardpoints[] = {"Splits_B_Missiles","B_MISSILE_PYLON"};
```

Comment:
nit—I don't *think* these should be case-sensitive and if they're not you can do this. Disregard if they are or you just don't feel like changing the repetition.
```suggestion
						hardpoints[] = {"Splits_B_Missiles","B_Missile_Pylon"};
```

Reply:
Everything I've gleaned about this silly system suggests sensitivity.... okay I can't alliterate anymore, I don't think they're case sensitive but it'll bother me if I don't standardize them <3